packages/x-date-pickers/src/PickersActionBar/PickersActionBar.tsx

```diff
@@ -22,10 +22,11 @@ export interface PickersActionBarProps {
   onClear: () => void;
   onCancel: () => void;
   onSetToday: () => void;
+  className?: string;
 }
 
 export const PickersActionBar = (props: PickersActionBarProps) => {
-  const { onAccept, onClear, onCancel, onSetToday, actions } = props;
+  const { onAccept, onClear, onCancel, onSetToday, actions, className } = props;
```

Comment:
To be more generic, you can extend `DialogActionProps` in the definition of `PickersActionBarProps` and then propagate all the other props to the `DialogActions`

```suggestion
  const { onAccept, onClear, onCancel, onSetToday, actions, ...other } = props;
```

Reply:
Thanks for a great suggestion 👍 🙇